ra-optimisation/rao-commons/src/test/java/com/farao_community/farao/rao_commons/linear_optimisation/fillers/MaxLoopFlowFillerTest.java

```diff
@@ -66,5 +77,53 @@ public void testFill() {
         MPVariable flowVariable = linearProblem.getFlowVariable(cnec1);
         assertEquals(1, loopFlowConstraintUb.getCoefficient(flowVariable), 0.1);
         assertEquals(1, loopFlowConstraintLb.getCoefficient(flowVariable), 0.1);
+
+        cnecLoopFlowExtension.setLoopflowShift(49.0);
+
+    }
+
+    @Test
+    public void testFillCaseInitialLoopFlowIsGreaterThanInputThreshold() {
+
+        CnecLoopFlowExtension cnecLoopFlowExtension = new CnecLoopFlowExtension(100.0, Unit.MEGAWATT);
+        cnecLoopFlowExtension.setLoopFlowConstraintInMW(100.0);
+        cnecLoopFlowExtension.setLoopflowShift(49.0);
+        cnec1.addExtension(CnecLoopFlowExtension.class, cnecLoopFlowExtension);
+        String testVariant = "test-variant";
+        ResultVariantManager resultVariantManager = new ResultVariantManager();
+        crac.addExtension(ResultVariantManager.class, resultVariantManager);
+        crac.getExtension(ResultVariantManager.class).createVariant(testVariant);
+        crac.getExtension(ResultVariantManager.class).setInitialVariantId(testVariant);
+        cnec1.getExtension(CnecResultExtension.class).getVariant(testVariant).setFlowInMW(95.);
+        initRaoData(crac.getPreventiveState());
+
+        boolean isLoopFlowApproximation = true; // currently cannot be tested without the loop-flow approximation, otherwise a sensitivity computation should be made
+        double loopFlowConstraintAdjustmentCoefficient = 5.;
+        double loopFlowAcceptableAugmentation = 13.;
+        double loopFlowViolationCost = 10.;
+        SensitivityAnalysisParameters sensitivityAnalysisParameters = new SensitivityAnalysisParameters();
+
+        MaxLoopFlowFiller maxLoopFlowFiller = new MaxLoopFlowFiller(isLoopFlowApproximation, loopFlowAcceptableAugmentation, loopFlowConstraintAdjustmentCoefficient, loopFlowViolationCost, sensitivityAnalysisParameters);
+
+        // build problem
+        coreProblemFiller.fill(raoData, linearProblem);
+        maxLoopFlowFiller.fill(raoData, linearProblem);
+
+        // check flow constraint for cnec1
+        MPConstraint loopFlowConstraintUb = linearProblem.getMaxLoopFlowConstraint(cnec1, LinearProblem.BoundExtension.UPPER_BOUND);
+        MPConstraint loopFlowConstraintLb = linearProblem.getMaxLoopFlowConstraint(cnec1, LinearProblem.BoundExtension.LOWER_BOUND);
+
+        assertNotNull(loopFlowConstraintUb);
+        assertNotNull(loopFlowConstraintLb);
+
+        assertEquals(-(95 + 13 - 5.) + 49.0, loopFlowConstraintLb.lb(), DOUBLE_TOLERANCE);
+        assertEquals((95 + 13 - 5.) + 49.0, loopFlowConstraintUb.ub(), DOUBLE_TOLERANCE);
```

Comment:
Actually now that the loopFlowAcceptableAugmentation is set directly in the `CnecLoopFlowExtension.loopFlowConstraintInMW` I'm surprised that it would work. I think we should expect (95 - 5) + 49 here.